src/components/FooterContainer/index.tsx

```diff
@@ -65,8 +45,33 @@ const Container = styled(Row)`
   }
 `
 
-const Footer = () => (
-  <>
+const Disclaimer = styled.div`
+  color: inherit;
+  min-height: ${themeSpacing(25)};
+  height: 100%;
+`
+
+const StyledHeading = styled(Heading)`
+  margin: 0 0 ${themeSpacing(3)} 0;
+  color: inherit;
+`
+
+const StyledFooterBottom = styled(FooterBottom)`
+  span {
+    font-family: Avenir Next LT W01 Demi, arial, sans-serif;
+    font-size: 14px;
+    font-weight: normal;
+    line-height: 22px;
```

Comment:
It should not be necessary to set these rules. What made it so that these rules were added?

Reply:
That is probably because I introduced the 'amsterdam-styled-components' footer component